examples/shader/post_processing.rs

```diff
@@ -20,13 +20,16 @@ use bevy::{
 };
 
 fn main() {
-    let mut app = App::new();
-    app.add_plugins(DefaultPlugins)
+    App::new()
+        .insert_resource(AssetServerSettings {
```

Comment:
I'm not sure this example actually needs this. It's not necessary to showcase post_processing. I think people should just add it manually if they want to play with it. Otherwise we might as well just add it to all examples with assets.

Same thing applies for the shader_instancing example